paloma/chain.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "paloma",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://palomachain.com/",
+  "pretty_name": "Paloma",
+  "chain_id": "tumbler",
+  "bech32_prefix": "paloma",
```

Comment:
Please register this prefix https://github.com/satoshilabs/slips/blob/master/slip-0173.md